src/ensembl/index.js

```diff
@@ -282,14 +284,16 @@ const uploadFile = async (opt) => {
         const gene = visited[geneId];
         const versionedGene = visited[key];
 
-        await conn.addRecord({
-            content: {
-                in: rid(versionedGene), out: rid(gene), source: rid(source),
-            },
-            existsOk: true,
-            fetchExisting: false,
-            target: 'generalizationof',
-        });
+        if (newGene || newGeneVersion) {
+            await conn.addRecord({
```

Comment:
the 2 genes could exists w/o this edge existing if they were loaded in a different way so it probably makes more sense to just continue to try to always create this for now